### PR TITLE
Fix security error message in wms filter

### PR DESCRIPTION
**Pull request by DelazJ**
refs #6082

### CODE REVIEW

File: docs/server_manual/services.rst [hunk 1]
```diff
@@ -1391,8 +1391,6 @@ GetFeatureInfo
 Standard parameters for the **GetFeatureInfo** request according to
 the OGC WMTS 1.0.0 specifications:
```
Comment (havatv) [inline docs/server_manual/services.rst:1393]:
```suggestion
the OGC WMTS 1.0.0 specification:

`WMTS 1.0.0 <http://portal.opengeospatial.org/files/?artifact_id=35326&format=pdf>`_

```

**Review by havatv — COMMENTED** [review]
I added the correct link to the WMTS 1.0.0 specification.

File: docs/server_manual/services.rst (hunk 1 above)
Comment (DelazJ) [reply docs/server_manual/services.rst:1393]: The link is unnecessary. Like any other service, it's already mentioned at the beginning of the wmts section. No need to repeat it for the requests.
And "specifications" does not necessarily look wrong to me.

Comment (havatv) [reply docs/server_manual/services.rst:1393]: > And "specifications" does not necessarily look wrong to me.

"specifications is" plural, and we are talking about only *one* specification (the WMTS specification), so the **s** must be removed.

Comment (DelazJ) [reply docs/server_manual/services.rst:1393]: I read it as "the WMTS standard has some specifications" (_rules you might follow to...._)

Comment (havatv) [reply docs/server_manual/services.rst:1393]: > I read it as "the WMTS standard has some specifications" (_rules you might follow to...._)

We are talking about **the** specification. "standard" and "specification" are used as synonyms.
In http://opengeospatial.github.io/e-learning/wmts/text/operations.html :

```Version 1.0.0 of the WMTS Specification (“WMTS Spec”) 07-057r7 OpenGIS Web Map Tile Service Implementation Standard was published in 2010```

**Pull request merged**